models/users.lua

```diff
@@ -3,21 +3,33 @@ local config   = require("lapis.config").get()
 local token = config.secret
 local trim = require("lapis.util").trim_filter
 local Model = require("lapis.db.model").Model
+local i18n = require("i18n")
 local Users = Model:extend("users", {
   relations = {
     {"packages", has_many = "Packages"}
   }
 })
 
+local function uuid()
```

Comment:
This seems scary...

Reply:
why? I could call it something else, but Lua doesn't have a uuid function and this generates something pretty close to one.